elastic/kibana
# [ResponseOps] rules runs recovered actions without ever running active actions

Resolves https://github.com/elastic/kibana/issues/182888

## Summary

This PR resolves a bug where rules will run the recovery actions for a delayed active alert.

### Checklist



### To verify

- Create a rule and set the `alert after x consecutive matches` to be greater than 1. It may be helpful for testing to include recovered and active actions.
- Make sure the delayed active alert recovers before hitting the consecutive matches threshold. 
- Verify that the rule does not send a recovery action and does not show a recovered alert in the rule run table.
